### PR TITLE
Study Article

### PR DESCRIPTION
**Pull request by GarthDB**

### CODE REVIEW

File: articles/_posts/2015-02-25-open-source-study.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Open Study"
+layout: post
+category: articles
+author: garthdb
+priority: 1
+tags: ux,study
+excerpt:
+---
+##Insufficient Data
```
Comment (una) [inline articles/_posts/2015-02-25-open-source-study.md]: "Insufficient" has a negative connotation. How about, "Gathering Data" or "Anecdotal Research"

File: articles/_posts/2015-02-25-open-source-study.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Open Study"
```
Comment (una) [inline articles/_posts/2015-02-25-open-source-study.md]: I would include "Open Study: A Call for Help" or something about the call to action in the title here

File: articles/_posts/2015-02-25-open-source-study.md (hunk 1 above)
Comment (GarthDB) [inline articles/_posts/2015-02-25-open-source-study.md]: how about "MOAR DATA!"

File: articles/_posts/2015-02-25-open-source-study.md (hunk 2 above)
Comment (GarthDB) [inline articles/_posts/2015-02-25-open-source-study.md]: Good call.

File: articles/_posts/2015-02-25-open-study.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Open Study: A Call for Help"
+layout: post
+category: articles
+author: garthdb
+priority: 1
+tags: ux,study
+excerpt:
+---
+##MOAR DATA!
```
Comment (una) [inline articles/_posts/2015-02-25-open-study.md]: :+1:

File: articles/_posts/2015-02-25-open-source-study.md (hunk 1 above)
Comment (winterdamsel) [inline articles/_posts/2015-02-25-open-source-study.md]: Or "Brainpicking"!

**Pull request merged**